azure-kit-demo/src/main/resources/application.yml

```diff
@@ -11,7 +11,7 @@ vaadin:
   azure:
     hazelcast:
       kubernetes:
-        service-name: azure-kit-hazelcast-service
+        xxxservice-name: azure-kit-hazelcast-service
```

Comment:
I suppose we still need this property to property configure Hazelcast for Kubernetes, right?

Reply:
My fault. I renamed the property to run the application locally and forgot to restore it.
Great catch!

Reply:
done